src/develop/imageop.h

```diff
@@ -569,7 +569,7 @@ gboolean dt_iop_have_required_input_format(const int required_ch,
                                            const dt_iop_roi_t *const roi_in,
                                            const dt_iop_roi_t *const roi_out);
 
-
+gboolean dt_iop_canvas_not_sensitive(const struct dt_develop_t *dev);
```

Comment:
I'll add a little comment here.

Reply:
Reading my comment I see that it is not really clear... I meant that a comment is needed here and not that I'll add it myself :)

Reply:
I only read quickly and indeed misunderstood, but makes more sense this way ;-)